### PR TITLE
Provide instructions on how to install previous versions

### PR DESCRIPTION
**Pull request by leezen**
Fixes https://github.com/pulumi/docs/issues/3090

### CODE REVIEW

File: content/docs/get-started/install/_index.md [hunk 1]
```diff
@@ -171,6 +171,66 @@ or visit https://pulumi.com/docs/reference/install/ for manual instructions and
...
+### Homebrew
+
+```bash
+$ brew install pulumi@<version>
+```
+
+### Installation Script
+
+```bash
+$ curl -fsSL https://get.pulumi.com | sh -s -- --version <version>
```
Comment (clstokes) [inline content/docs/get-started/install/_index.md:191]: The double dashes (`--`) get rendered as single dashes (-) when published and as such don't work when copy/pasted and used as single dashes.

from http://pulumi-docs-origin-pr-4200-23d76ac1.s3-website.us-west-2.amazonaws.com/docs/get-started/install/#installing-previous-versions:

image

Comment (clstokes) [reply content/docs/get-started/install/_index.md:191]: CLI output with single dashes and double dashes

```
root@f69176c035e3:/# curl -fsSL https://get.pulumi.com | sh -s - -version 2.2.0
=== Installing Pulumi v2.10.0 ===
+ Downloading https://get.pulumi.com/releases/sdk/pulumi-v2.10.0-linux-x64.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
 72 61.2M   72 44.1M    0     0  11.8M      0  0:00:05  0:00:03  0:00:02 11.8M^C
root@f69176c035e3:/# curl -fsSL https://get.pulumi.com | sh -s -- --version 2.2.0
=== Installing Pulumi v2.2.0 ===
+ Downloading https://get.pulumi.com/releases/sdk/pulumi-v2.2.0-linux-x64.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 57.2M  100 57.2M    0     0  7337k      0  0:00:07  0:00:07 --:--:-- 7043k
+ Extracting to /root/.pulumi/bin

=== Pulumi is now installed! 🍹 ===
+ Please add /root/.pulumi/bin to your $PATH
+ Get started with Pulumi: https://www.pulumi.com/docs/quickstart
```

**Review by clstokes — CHANGES_REQUESTED** [review]
Content looks great but the double dash problem causes the commands to not work.

File: content/docs/get-started/install/_index.md [hunk 2]
```diff
@@ -171,6 +171,66 @@ or visit https://pulumi.com/docs/reference/install/ for manual instructions and
...
+{{% /choosable %}}
+
+{{% choosable os linux %}}
+
+### Installation Script
+
+To install, run our installation script:
+
+```bash
+$ curl -fsSL https://get.pulumi.com | sh -s -- --version <version>
```
Comment (clstokes) [inline content/docs/get-started/install/_index.md:203]: Same double-dash to single-dash rendering problem.

**Comment by leezen** [comment]
@cnunciato Thanks for the helpful fix!

**Review by cnunciato — APPROVED** [review]

**Review by clstokes — APPROVED** [review]
lgtm

**Pull request merged**